atlascope/core/management/populate/embeddings.json

```diff
@@ -1,16 +1,16 @@
 [
     {
-        "investigation": "Whole Team Investigation",
-        "id": 301,
-        "parent": "Remote Vandy Dataset",
-        "child": "Local Vandy Dataset",
+        "investigation": "Semantic zoom playground",
+        "id": 3000,
+        "parent": "HTA9_1_BA_F_ROI02",
+        "child": "HTA9_1_BA_F_ROI04",
         "child_bounding_box": [100, 100, 200, 200]
     },
     {
-        "investigation": "Whole Team Investigation",
-        "id": 302,
-        "parent": "Local Vandy Dataset",
-        "child": "Local Vandy Dataset (Copy)",
-        "child_bounding_box": [100, 100, 200, 200]
+        "investigation": "Semantic zoom playground",
+        "id": 3001,
+        "parent": "HTA9_1_BA_F_ROI02",
+        "child": "HTA9_1_BA_F_ROI02 (Copy)",
```

Comment:
I think for the purpose of the Semantic zoom playground investigation, we should avoid putting the same image inside itself. Perhaps put dataset B inside dataset A in two different ways